local-pv-provisioner/Makefile

```diff
@@ -45,7 +40,7 @@ generate:
 check-uncommitted:
 	$(MAKE) manifests
```

Comment:
Can the `manifests` and `generate` targets be written as the dependent target, like `check-uncommitted: manifests generate`??

Reply:
Yes. They can be written as the dependent target.
But I don't change it because it makes the intentions clearer, I think.